src/contrastEnhancement.h

```diff
@@ -0,0 +1,31 @@
+
+#ifndef OPENFLEXURE_CONTRAST_ENHANCEMENT_H
+#define OPENFLEXURE_CONTRAST_ENHANCEMENT_H
+
+
+#include <opencv2/core.hpp>
+#include <opencv2/videoio.hpp>
+#include <iostream>
+#include <stdlib.h>
+#include <thread>
+#include "imageProcessor.h"
+
+
+class contrastEnhancement:   public imageProcessor{
+   
+
+public:
+    contrastEnhancement() = default;
+    int threshold= 0;
+    int sliderThreshold = 10; //slider threshold used for metadata purposes
+    void receiveFrame(frame newFrame);
+    void updateThreshold(int value);
```

Comment:
make private along with threshold and sliderthreshold

Reply:
I cant make them private since they are used in gui.cpp